lua/sql.lua

```diff
@@ -385,19 +287,21 @@ function sql:add(...) return sql:insert(...) end
 ---@usage db:update{ "todos" = { where = { deadline = "2021" }, values = { status = "overdue" }}}
 ---@todo support unnamed or anonymous args
 function sql:update(...)
+  local method = "update"
   local args = {...}
   local ret_vals = {}
 
   local inner_eval = function(tbl, p)
-    return self:eval(self:parse("update", {
-      tbl = tbl,
+    local sqlstmt = parse(tbl, method, {
       set = p.values,
       where = p.where,
-      placeholders = true,
-    }), u.tbl_extend('force', p.values, p.where))
+      named = true,
+    })
+    return self:eval(sqlstmt, u.tbl_extend('force', p.values))
```

Comment:
what are you extending? don't you need to extend with where. If not just remove the tbl_extend

Reply:
oh yah nice catch @Conni2461. where is handled by the parser from now on